generator/src/lib.rs

```diff
@@ -991,21 +997,14 @@ fn generate_function_pointers<'a>(
                 .as_ref()
                 .expect("Command must have return type");
 
-            let type_in_module = fn_cache.get(name.as_str()).cloned();
+            let define_pfn = !fn_cache.contains(name.as_str());
 
-            if let Some((vendor, ident)) = module {
-                if type_in_module.is_none() {
-                    fn_cache.insert(
-                        name,
-                        (format_ident!("{}", vendor.to_lowercase()), ident.clone()),
-                    );
-                }
+            if define_pfn {
+                fn_cache.insert(name);
             }
 
             Command {
-                // PFN function pointers are global and can not have duplicates.
-                // This can happen because there are aliases to commands
```

Comment:
We should probably keep tracking with a comment somewhere that because of aliases some loaders may overlap and load similar functions to others.

Reply:
Restored similar wording.

Reply:
Thanks! In hindsight it's not even (just) aliases if I remember correctly, but multiple `extensions` (including the base v1.x Vulkan core spec) `<require>`ing that `PFN_` type, and us naively emitting the `type PFN_` on the first occurrence while using a `HashSet` to prevent it from being emitted again.